config/passport.js

```diff
@@ -1,31 +1,60 @@
 import passport from 'passport';
-import bcrypt from 'bcrypt';
 import local from 'passport-local';
-import db from '../models';
+import GoogleStrategy from 'passport-google-oauth20';
+import FacebookStrategy from 'passport-facebook';
+import AuthController from '../controllers/authController';
+import User from '../models/User';
 
 const LocalStrategy = local.Strategy;
 
-passport.use(new LocalStrategy(
-  {
-    usernameField: 'user[email]',
-    passwordField: 'user[password]'
-  },
-  ((email, password, done) => {
-    db.User.findOne({ where: { email } })
-      .then((user) => {
-        bcrypt.compare(
-          password,
-          user.hashedPassword,
-          (err, res) => {
-            if (!user || !res || err) {
-              return done(null, false, {
-                errors: { 'email or password': 'is invalid' }
-              });
-            }
-            return done(null, user);
+const passportConfig = (app) => {
+  app.use(passport.initialize());
+  app.use(passport.session());
+  passport.use(new LocalStrategy(
+    {
+      usernameField: 'user[email]',
+      passwordField: 'user[password]'
+    },
+    ((email, password, done) => {
+      User.findOne({ email })
```

Comment:
I'll suggest you use object destructing from there doing something ;
`import { User } from '../models'`;

In case you might need to get more objects/models instead of importing twice or more.

Reply:
Thanks, All suggestions have been implemented.

Reply:
Thanks, this has been fixed